### DISCUSSION TITLE
Option to disable default notification channel

### DISCUSSION THREAD

**Discussion by olivierboudet** [grafana/grafana#41645] [Alerting]
I am using a default notification channel to garantuee that all my alerts have at least one channel defined.
On some specific alerts, I do not want this default channel to be activated, only an other.

Can we have an option to remove the default channel on an alert when others notification channel are also defined ?

I am open to contribute this if it seems reasonable.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
